app/models/case_contact.rb

```diff
@@ -25,6 +25,9 @@ class CaseContact < ApplicationRecord
     supervisor
     therapist
     youth
+    medical_professional
+    dss_worker
```

Comment:
When the decorator is used to display the dss_worker type, what will it look like? Will it be "DSS Professional" or "Dss Professional"? If it is "Dss Professional", will that be acceptable?

https://github.com/rubyforgood/casa/blob/master/app/decorators/case_contact_decorator.rb#L43-L47

Reply:
We do want "DSS Worker", we should add a UI prettifier somehow also